main.tf

```diff
@@ -51,105 +55,49 @@ resource "aws_iam_role" "this" {
   tags = var.tags
 }
 
-resource "aws_iam_role_policy_attachment" "assume_roles_attach" {
-  policy_arn = aws_iam_policy.assume_roles.arn
-  role       = aws_iam_role.this.name
-}
-
-# Allow the runtime to assume roles in the /sym/ path in safelisted accounts
 resource "aws_iam_policy" "assume_roles" {
   name = local.role_name
   path = "/sym/"
 
-  description = "Base permissions for the Sym runtime"
+  description = "These are base permissions required for the Sym Runtime to perform any actions in your AWS account. This policy allows the Sym Runtime to assume roles in the /sym/ path in the current AWS account."
   policy = jsonencode({
     Statement = [{
       Action   = "sts:AssumeRole"
       Effect   = "Allow"
-      Resource = local.target_resources
+      Resource = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/sym/*"]
```

Comment:
We still need the account ID safelist so that the runtime can assume roles in other accounts, like if you have your runtime in one account but want to assume the sso connector role in the sso management account.

Reply:
The policy that allows you to assume roles in the SSO account will be generated separately and attached to the runtime_connector_role, which is why we decided not to have the account ID safelist here.

Reply:
- For symflow generate, we won't worry about using the `account_id_safelist`
- But for using the module manually, such as in the multi-tenant SSO strategy example, it's cleaner to pass in all the account IDs instead of generating the boiler plate to pass in

Reply:
Got it... and just to make sure I'm understanding, the boilerplate would have been extra policies and attachments for the other accounts?

Reply:
https://github.com/symopsio/examples/blob/main/advanced/multi_tenant_sso_strategy/multi_tenant_aws_sso.tf#L22

Yeah, so instead of generating a duplicate policy, you would just pass in the additional account IDs to the module

Reply:
Okay cool, thanks for the explanation!